[2024-03-03T03:41:53.776+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_data_files.get_files_3 scheduled__2016-12-12T00:00:00+00:00 [queued]>
[2024-03-03T03:41:53.792+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_data_files.get_files_3 scheduled__2016-12-12T00:00:00+00:00 [queued]>
[2024-03-03T03:41:53.793+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-03T03:41:53.823+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): get_files_3> on 2016-12-12 00:00:00+00:00
[2024-03-03T03:41:53.834+0000] {standard_task_runner.py:60} INFO - Started process 4601 to run task
[2024-03-03T03:41:53.840+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_data_files', 'get_files_3', 'scheduled__2016-12-12T00:00:00+00:00', '--job-id', '2098', '--raw', '--subdir', 'DAGS_FOLDER/dummy_dag.py', '--cfg-path', '/tmp/tmpnwif4nod']
[2024-03-03T03:41:53.844+0000] {standard_task_runner.py:88} INFO - Job 2098: Subtask get_files_3
[2024-03-03T03:41:53.867+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-03T03:41:53.927+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_data_files.get_files_3 scheduled__2016-12-12T00:00:00+00:00 [running]> on host 716a53fa82e1
[2024-03-03T03:41:54.072+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_data_files' AIRFLOW_CTX_TASK_ID='get_files_3' AIRFLOW_CTX_EXECUTION_DATE='2016-12-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2016-12-12T00:00:00+00:00'
[2024-03-03T03:41:56.058+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-03T03:41:56.075+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_data_files, task_id=get_files_3, execution_date=20161212T000000, start_date=20240303T034153, end_date=20240303T034156
[2024-03-03T03:41:56.107+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-03T03:41:56.131+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
